test/compatibility/config_test.lua

```diff
@@ -71,14 +71,12 @@ function g.test_failover_2_0_1_78()
         cluster { failover_params(mode: "eventual") {
             mode
             state_provider
-            tarantool_params {}
         }}
     }]]}).data.cluster.failover_params
 
     t.assert_equals(failover_params, {
         mode = 'eventual',
         state_provider = box.NULL,
-        tarantool_params = box.NULL,
```

Comment:
I suggest to keep it, but change to `tarantool_params = {}`

You may also add `etcd2_params {}` in query